### ISSUE TITLE
Module java-security-test:2.7.5 references vulnerable jetty server version

### ISSUE THREAD

**Issue by Emdee89** [SAP/cloud-security-xsuaa-integration#350]
Maven dependency https://mvnrepository.com/artifact/com.sap.cloud.security/java-security-test/2.7.5 references several dependencies from the realm of jetty server (example: https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-server) in version `9.4.24.v20191120`(example: https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-server/9.4.24.v20191120).

Our security tool found that the mentioned jetty server versions are vulnerable, see here: https://nvd.nist.gov/vuln/detail/CVE-2019-17638

Requirement: Update your dependencies to use the latest version `9.4.30.v20200611` (example: https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-server/9.4.30.v20200611)

**Comment by hassler-d** [comment]
Hello,

thanks for the input. I will take a look at it. I hope you are not using java-security-test in production since it is only intended for tests :)

Best Regards
Daniel

**Comment by hassler-d** [comment]
Hi again,

I have looked at the issue. We cannot easily upgrade our jetty dependency because we also rely on wiremock in our test package. The problem is that wiremock depends on an older version of jetty. This older version is not compatible with the version you suggested. If we try to upgrade we run into `IncompatibleClassChange` errors. 

I guess we blocked here until wiremock upgrades their jetty dependency.

Best Regards
Daniel
